PSICover/PSICover.cs

```diff
@@ -164,18 +169,47 @@ class Analyzer {
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
          foreach (var block in blocks) {
+            bID = hits[block.Id];
             bool hit = hits[block.Id] > 0;
-            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\">";
-            code[block.ELine] = code[block.ELine].Insert (block.ECol, "</span>");
-            code[block.SLine] = code[block.SLine].Insert (block.SCol, tag);
+            string tag = $"<span class=\"{(hit ? "hit tooltip" : "unhit")}\">";
+            if (block.ELine == block.SLine) {
+               code[block.ELine] = code[block.ELine].Insert (block.ECol, $"<span class=\"tooltiptext\">No. of hits = {bID}</span></span>");
+               code[block.SLine] = code[block.SLine].Insert (block.SCol, tag);
+            } else {
+               for (int l = block.ELine; l <= block.SLine; l--) {
+                  code[l] = code[block.ELine].Insert (block.ECol, $"<span class=\"tooltiptext\">No. of hits = {bID}</span></span>");
+                  code[l] = code[block.SLine].Insert (block.SCol, tag);

```

Comment:
block.ELine will be always greater than or equal to block.SLine. So, this loop is not getting executed. It is always better to use breakpoint in newly written code to see how it behaves.